client/main/ribbon.js

```diff
@@ -125,8 +125,14 @@ const RibbonView = Backbone.View.extend({
         this.$tabs = this.$header.find('.jmv-ribbon-tab');
 
         this.appMenu = new AppMenu({ el: this.$appMenu, model: this.model });
-        this.appMenu.on('shown', event => this._menuShown(event));
-        this.appMenu.on('hidden', event => this._closeMenus());
+        this.appMenu.on('shown', event => {
+            this._menuShown(event);
+            $('#header-buttons').hide();
+        });
+        this.appMenu.on('hidden', event => {
+            this._closeMenus();
+            $('#header-buttons').show();
```

Comment:
I also don't really like the solution i am putting forward here, but i couldn't get the z-levels to work, so i resorted to this hack so that the new options panel would at least work properly on windows machines. If you can get it to work with z-levels then awesome.